packages/modal/src/styles/Modal.module.css

```diff
@@ -14,13 +14,13 @@
 .overlay {
   position: fixed;
   inset: 0;
-  background-color: hsl(0 0% 20% / 0.5);
+  background-color: rgb(40 40 40 / 0.5);
   display: flex;
   align-items: flex-end;
   justify-content: center;
 
   :root[data-theme='dark'] & {
-    background-color: rgb(18 18 18 / 0.75);
+    background-color: rgb(25 25 25 / 0.75);
```

Comment:
Just to confirm: the default theme backgrounds are the same for drawer/modal, but the dark them has a slightly different alpha value (`0.75` here in the modal, vs `0.7` in the drawer)?

Reply:
Good catch! Fixing that

Reply:
Fixed.